packages/core/src/tracing/spanAttributes.ts

```diff
@@ -0,0 +1,4 @@
+export const SentrySemanticAttributes = {
+  Source: 'sentry.source',
+  SampleRate: 'sentry.sample_rate',
+} as const;
```

Comment:
m: I have a slight concern here. This object is not tree shakable, in the sense that as soon as it grows, whenever you use one value, the entire object is pulled into the bundle.

I would suggest we either a) expose the attribute consts as individual variables, b) create individual variables and put it under some name space that is still tree-shakable.

Reply:
Yeah, I was also not 100% sure. I modeled this after OTEL, which does it the same way.  🤔 

We could do:

```ts
export const SENTRY_ATTR_SOURCE = 'sentry.source';
export const SENTRY_ATTR_SAMPLE_RATE = 'sentry.sample_rate';
```

?

Reply:
I would probably just do `SEMANTIC_ATTRIBUTE_SENTRY_SOURCE`. Abbreviations feel weird.

Reply:
Otel probably didn't consider this because they don't think too much about browser land I imagine. We will have to consider this.